content/shmem_alltoall.tex

```diff
@@ -103,7 +103,7 @@
     This routine restores \VAR{pSync} to its original contents.  Multiple calls
     to \openshmem\ routines that use the same \VAR{pSync} array do not require
     that \VAR{pSync} be reinitialized after the first call.
-    You must ensure that the \VAR{pSync} array is not being updated by any
+    The user must ensure that the \VAR{pSync} array is not being updated by any
```

Comment:
I'm not sure that a macro would work for this because we multiple phrasings involving "user".  The best solution is probably to allow Mike to make the style consistent and then do our best to follow it.